libstuff/libstuff.h

```diff
@@ -246,14 +250,42 @@ extern atomic<void (*)(int priority, const char *format, ...)> SSyslogFunc;
     } while (false)
 
 #define SLOGPREFIX ""
-#define SDEBUG(_MSG_) SSYSLOG(LOG_DEBUG, "[dbug] " << SLOGPREFIX << _MSG_)
-#define SINFO(_MSG_) SSYSLOG(LOG_INFO, "[info] " << SLOGPREFIX << _MSG_)
-#define SHMMM(_MSG_) SSYSLOG(LOG_NOTICE, "[hmmm] " << SLOGPREFIX << _MSG_)
-#define SWARN(_MSG_) SSYSLOG(LOG_WARNING, "[warn] " << SLOGPREFIX << _MSG_)
-#define SALERT(_MSG_) SSYSLOG(LOG_ALERT, "[alrt] " << SLOGPREFIX << _MSG_)
+#define SDEBUG(_MSG_, ...)                          \
+    do {                                            \
+        stringstream ss;                            \
+        ss << "[dbug] " << SLOGPREFIX << _MSG_;     \
+        SSYSLOG(LOG_INFO, ss.str(), ##__VA_ARGS__); \
+    } while (false)
+#define SINFO(_MSG_, ...)                           \
+    do {                                            \
+        stringstream ss;                            \
+        ss << "[info] " << SLOGPREFIX << _MSG_;     \
```

Comment:
Why did we take these two lines out of SSYSLOG just to repeat them a bunch of times here?

Reply:
It did not compile otherwise 😕